[Funnel P0] add PretrainedConfig and unit test

robinbg opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/PaddleNLP/pull/26 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ Models | APIs | Docs | Others ] -->

### Description
<!-- Describe what this PR does -->

robinbg committed:
Create configuration.py

robinbg committed:
Rename paddlenlp/transformers/fnet/configuration.py to paddlenlp/transformers/funnel/configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update __init__.py

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleNLP?pullRequest=4939) <br/>All committers have signed the CLA.

sijunhe reviewed (commented):
您好，欢迎参加飞桨黑客松。这个PR是一个好的开始，后续需要写好test_modeling和test_tokenizer单测，并且做到单测通过

robinbg committed:
Update modeling.py

robinbg committed:
Update configuration.py

robinbg committed:
Update modeling.py

robinbg committed:
Update modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	new file:   tests/transformers/funnel/__init__.py
	new file:   tests/transformers/funnel/test_modeling.py
	new file:   tests/transformers/funnel/test_tokenizer.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py
	modified:   tests/transformers/funnel/test_tokenizer.py

robinbg committed:
Update test_modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:

robinbg committed:
On branch patch-
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/configuration.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
Merge branch 'patch-4' of https://github.com/robinbg/PaddleNLP into patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
Changes to be committed:
	modified:   tests/transformers/test_modeling_common.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/test_modeling_common.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/test_modeling_common.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/test_modeling_common.py

robinbg committed:
On branch patch-4
 Changes to be committed:
:
:

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   paddlenlp/transformers/funnel/modeling.py

robinbg committed:
114

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg committed:
Your branch is up to date with 'origin/patch-4'.

robinbg committed:
da:

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4

robinbg committed:
aOn branch patch-4

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg committed:
On branch patch-4

robinbg committed:
Changes to be committed:
	modified:   tests/transformers/funnel/test_modeling.py

robinbg committed:
On branch patch-4

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.

robinbg commented:
您好，请问这里找不到pretrained_resource_files_map是什么原因啊，明明在FunnelTokenizer里有这个attribute啊

sijunhe commented on tests/transformers/funnel/test_tokenizer.py:
```diff
@@ -0,0 +1,75 @@
+import os
+import unittest
+
+from transformers import FunnelTokenizer
```
这里应该是paddlenlp.transformers

robinbg replied:
好的，谢谢

sijunhe reviewed (commented):
你遇到的问题原因是加载的tokenizer是hf的，而不是paddlenlp的。同时我建议你不要完全依赖我们的ci来开发，效率会很低。如果本地没有开发环境，可以免费使用github的codespace

robinbg committed:
modified:   tests/transformers/funnel/test_tokenizer.py

robinbg committed:
On branch patch-4
 Your branch is up to date with 'origin/patch-4'.
 Changes to be committed:
	modified:   tests/transformers/funnel/test_tokenizer.py

robinbg committed:
Merge branch 'patch-4' of https://github.com/robinbg/PaddleNLP into patch-4

robinbg committed:
Changes to be committed:
	modified:   tests/transformers/funnel/test_tokenizer.py
